homeworks/david.tereshchenko_DavidTereshchenko/GameProject/css/style.css

```diff
@@ -103,9 +104,10 @@ body {
 }
 
 .header__body {
-  display: flex;
+  display: flex; /* на мою жумку тут не потрібно робити грід */
```

Comment:
Comments in the code are not for communication. You can comment to separate blocks of code by meaning, or TODO for the future. Please remove this

Reply:
oou, sorry, forgot this